Missionframework/modules/23_logistic/functions.hpp

```diff
@@ -0,0 +1,62 @@
+/*
+    KP LIBERATION MODULE FUNCTIONS
+
+    File: functions.hpp
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2019-01-16
+    Last Update: 2019-02-24
+    License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
+
+    Description:
+        Defines for all functions, which are brought by this module.
+*/
+
+class logi {
```

Comment:
Named it logistic first and changed it later because of those long terms (permission -> perm) but maybe you're right and we should keep it like it is.